.github/workflows/trunk-artifacts.yml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2025, Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# These jobs build ISPC for trunk on every push to main pushing artifacts to
+# Github pre-release.
+name: Trunk Artifacts
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+env:
+  LLVM_REPO: https://github.com/ispc/ispc.dependencies
+  ISPC_HOME: ${{ github.workspace }}
+  LLVM_HOME: ${{ github.workspace }}
+  LLVM_VERSION: "18.1"
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-22.04
+    # Disabling this rebuild for non ispc/ispc repo
+    # TODO! uncomment before merge
+    # if: github.repository == 'ispc/ispc'
+    permissions:
+      contents: write  # Needed for release creation/update
+
+    env:
+      LLVM_TAR: llvm-18.1.8-ubuntu22.04-Release+Asserts-x86.arm.wasm.tar.xz
+
+    steps:
+    - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+      with:
+        submodules: false
+
+    - name: Install Dependencies
+      run: |
+        .github/workflows/scripts/install-build-deps.sh
+
+    - name: Configure
+      run: |
+        # Setup emcc for WASM target
+        source scripts/install_emscripten.sh && emcc --version
+        cmake -B build ./ -DISPC_PREPARE_PACKAGE=ON -DCMAKE_INSTALL_PREFIX="$ISPC_HOME"/install -DISPC_CROSS=ON -DWASM_ENABLED=ON -DCMAKE_CXX_FLAGS="-Werror"
```

Comment:
I think we should run here the command closest to our release packages (from linux-build-ispc-llvm18-release job?). Same for Windows.

Reply:
I have fixed it